docs/conf.py

```diff
@@ -61,7 +61,8 @@
     'reno.sphinxext',
     'nbsphinx',
     'sphinx_design',
-    "qiskit_sphinx_theme",
+    'sphinxext.rediraffe',
```

Comment:
Can you use https://pypi.org/project/sphinx-reredirects/ instead? We're using it already for Qiskit and it's nice to be consistent. We'll be adding a lot more redirects for https://github.com/Qiskit/documentation/issues/216

Reply:
Yep, done!